src/Symfony/Component/Process/Process.php

```diff
@@ -274,24 +277,23 @@ public function start(callable $callback = null)
             if ('\\' === DIRECTORY_SEPARATOR && !empty($this->options['bypass_shell']) && !$this->enhanceWindowsCompatibility) {
                 throw new LogicException('The "bypass_shell" option must be false to inherit environment variables while enhanced Windows compatibility is off');
             }
+            $escape = '\\' === DIRECTORY_SEPARATOR ? function ($v) { return ProcessUtils::escapeArgument($v, ProcessUtils::ESC_WINDOWS_CMD); } : 'escapeshellarg';
```

Comment:
and why applying on the CMD escaping, not the ARGV escaping, while Unix uses ``escapeshellarg`` ?